File: SuperGrid/C#/DemoGridArchitecture.cs
using DevComponents.DotNetBar;

namespace SuperGridDemo
{
    public partial class DemoGridArchitecture : Office2007Form
    {
        public DemoGridArchitecture()
        {
            InitializeComponent();

            // Set the sample description text

            ShellServices.LoadRtbText(richTextBox1,
                "SuperGridDemo.Resources.DemoArchitecture.rtf");
        }
    }
}

File: SuperGrid/VB/DemoGridArchitecture.vb
Namespace SuperGridDemo
	Partial Public Class DemoGridArchitecture
        Inherits DevComponents.DotNetBar.Office2007Form
		Public Sub New()
			InitializeComponent()

			' Set the sample description text

			ShellServices.LoadRtbText(richTextBox1, "SuperGridDemo.DemoArchitecture.rtf")
		End Sub
	End Class
End Namespace